docs/agent/missing_features.md

```diff
@@ -1,6 +1,6 @@
 # Missing Features
 
 This information is available in the Datadog documentation:<br>
-[docs.datadoghq.com/agent/faq/agent-v6-changes][1]
+[docs.datadoghq.com/agent/versions][1]
 
-[1]: https://docs.datadoghq.com/agent/faq/agent-v6-changes/#features
+[1]: https://docs.datadoghq.com/agent/versions/#features
```

Comment:
I believe the link should be:

```suggestion
[1]: https://docs.datadoghq.com/agent/faq/agent_v6_changes/?tab=agent#features
```

Reply:
At first I think this document would be for A6 / A7, hence the link to a more generic documentation page. Adding this to the change.